benchmarks/ampi/alltoall/Makefile

```diff
@@ -35,8 +35,8 @@ alltoall_VPtest: alltoall_VPtest.c
 	$(CHARMC) -o alltoall_VPtest alltoall_VPtest.o $(OPTS) $(LIBS)
 
 test: all
-	$(call run, +p4 ./mpibench +vp4)
-	$(call run, +p4 ./mpibench +vp8)
+#	$(call run, +p4 ./mpibench +vp4)
+#	$(call run, +p4 ./mpibench +vp8)
```

Comment:
Can you add comments why these are disabled?

Reply:
Done